core/templates/dev/head/pages/exploration_editor/editor_tab/customize_interaction_modal_directive.html

```diff
@@ -29,12 +29,18 @@ <h3 ng-if="stateInteractionIdService.displayed && (customizationModalReopened ||
 
   <form ng-if="stateInteractionIdService.displayed && hasCustomizationArgs" name="form.schemaForm" class="protractor-test-interaction-editor">
     <div ng-repeat="customizationArgSpec in customizationArgSpecs track by $index">
-      <div class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
-      <div>
+      <div ng-if="customizationArgSpec.schema.type!='bool'" class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
+      <div ng-if="customizationArgSpec.schema.type!='bool'">
         <schema-based-editor local-value="stateCustomizationArgsService.displayed[customizationArgSpec.name].value" schema="customizationArgSpec.schema">
         </schema-based-editor>
         <div style="margin-bottom: 15px;"></div>
       </div>
+      <div ng-if="customizationArgSpec.schema.type=='bool'" class="oppia-interaction-customization-label">
```

Comment:
Ditto: === instead of ==; put spaces on left and right of ==.

Reply:
okay.I will make those changes,thanks for the review.